renovate.json

```diff
@@ -69,7 +69,8 @@
       "matchSourceUrls": [
         "https://github.com/open-telemetry/opentelemetry-go-contrib"
       ],
-      "groupName": "All opentelemetry-go-contrib packages"
+      "groupName": "All opentelemetry-go-contrib packages",
+      "postUpdateOptions": ["gomodTidy"]
```

Comment:
does `gomodTidy` support repos w/ multiple go modules? the tidy-dependencies workflow was created to run mod tidy in all go modules in the repo

Reply:
That's a good question! I'm not totally sure since the docs are a bit vague, but my read of them is that these `postUpdateOptions` happen once per changed go.mod file.

Reply:
it would be great if you were able to confirm this before moving forward w/ the pr. If you're ok w/ separating this out of this PR, we could get the permissions changes in sooner

Reply:
👍 I'll confirm this in a separate repo and relay back what I find!

Definitely not opposed to opening the renovate + tidy-dependencies stuff as a separate PR. I'll work on splitting them apart this afternoon.